string name
string bond_id
